src/compass/utils/helpers.py

```diff
@@ -253,25 +256,29 @@ def get_burst_bbox(burst_id, burst_db_file=None, burst_db_conn=None):
         burst_db_conn = sqlite3.connect(burst_db_file)
     burst_db_conn.row_factory = sqlite3.Row  # return rows as dicts
 
-    burst_ids = [burst_id] if isinstance(burst_id, str) else burst_id
+    burst_ids = [burst_id] if isinstance(burst_id, str) else \
+        ','.join(burst_id)
 
-    results = []
-    query = "SELECT epsg, xmin, ymin, xmax, ymax FROM burst_id_map WHERE burst_id_jpl = ?"
-    for bid in burst_ids:
-        cur = burst_db_conn.execute(query, (bid,))
-        results.append(cur.fetchone())
+    query = "SELECT epsg, xmin, ymin, xmax, ymax burst_id_jpl FROM burst_id_map WHERE burst_id_jpl = ?"
```

Comment:
```suggestion
    query = "SELECT epsg, xmin, ymin, xmax, ymax FROM burst_id_map WHERE burst_id_jpl = ?"
```

as it stands, it's renaming `ymax` to `burst_id_jpl`, since it's equivalent to `SELECT epsg, ...  ymax AS burst_id_jpl FROM ...`